package.json

```diff
@@ -6,10 +6,50 @@
     "node": "10.16.0"
   },
   "dependencies": {
-    "@babel/core": "7.5.5",
     "@material-ui/core": "^4.2.1",
     "@material-ui/icons": "^4.2.1",
-    "@svgr/webpack": "4.3.2",
+    "dotenv": "6.2.0",
+    "dotenv-expand": "4.2.0",
```

Comment:
What is this for?

Reply:
well, you know dotenv, dotenv expand allows it to expand variables like:

```
VAR1=asd
VAR2=$VAR1+asd
```

It's not being used right now, but it came via CRA so I am not sure if it might be used internally

Reply:
Sure, was just looking for more cleanup targets